### PR TITLE
feat: Add getPhoneCodesForWilaya

### LINKED ISSUE #19: Feature request: Add `getPhoneCodeForWilaya` and `getPhoneCodesForWilaya `
⚠️⚠️⚠️  
**This issue is for first-timers only, we'd like you to start your open-source journey from here 😄**
⚠️⚠️⚠️

As a library consumer, I would like to have a method that returns phone codes for each wilaya.  
One of the use cases for this feature is with forms; in order to have a better user experience, we would like to set the phone code initials when the user select their wilaya. For example: if I fill the address block and set my city to Oran (code: 31) I would like if the phone number block is set automatically to 041.

What we want to add:
- A `getPhoneCodeForWilaya` function that takes one parameter (wilayaCode: number) and returns the first phone code (as a wilaya, like Algiers have two phone codes: 21, 23).
- A `getPhoneCodesForWilaya`  function that takes one parameter (wilayaCode: number) and returns an array of all phoneCodes of that wilaya.

**NB:**

- Please read our [contributing guidelines](https://github.com/dzcode-io/leblad/blob/develop/.github/CONTRIBUTING.md) and our [code of conduct](https://github.com/dzcode-io/leblad/blob/develop/.github/CODE_OF_CONDUCT.md)
- Comment here so we know that you're on it 🐝 .
- Clone this repository, read the code and try to understand how we structured it.
- Start hacking (_PS: You can check the [PRs we merged](https://github.com/dzcode-io/leblad/pulls?q=is%3Apr+is%3Amerged) and see how our smart contributors added this kind of features_)
- Update documentation in the readme file
- Ask for help, here or in [Slack](https://join.slack.com/t/dzcode/shared_invite/zt-ek9kscb7-m8z_~cBjX79l~uchuABPFQ).

Happy Hacking !

### PR DESCRIPTION
**Pull request by joeylnguyen**
# Description

- Added getPhoneCodesForWilaya method that returns an array of phone codes for a given wilaya code
- Added tests for getPhoneCodesForWilaya
- Exported method in index.js
- Updated tests for index.js
- Updated README

References #19 

# Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] I checked that there's no dataset update (can be done by running `npm run update-dataset`)
- [x] `npm test` passes on my machine
- [x] `npm run lint` passes on my machine

### CODE REVIEW

**Comment by joeylnguyen** [comment]
@Fcmam5 I plan on submitting a separate PR for getPhoneCodeForWilaya. Do you agree with that, or would you rather I add that feature to this PR?

File: README.md [hunk 1]
```diff
@@ -144,6 +144,22 @@ const { getWilayaByPhoneCode } = require('@dzcode-io/leblad');
 console.log(getWilayaByPhoneCode(34)); //will the wilaya object ({name: "Béjaïa"...})
 ```
 
+#### getPhoneCodesForWilaya(wilayaCode?: number)
```
Comment (Fcmam5) [inline README.md]: This field is required

```suggestion
#### getPhoneCodesForWilaya(wilayaCode: number)
```

**Review by Fcmam5 — COMMENTED** [review]
Thank you @joeylnguyen ! I just have one tiny suggestion 😄 .

**Comment by Fcmam5** [comment]
> @Fcmam5 I plan on submitting a separate PR for getPhoneCodeForWilaya. Do you agree with that, or would you rather I add that feature to this PR?

@joeylnguyen It's fine to submit it in a separate PR 😄 , this will make review easier for us.  
Let's just mention that they're both related #19  

Happy Hacking !

File: README.md (hunk 1 above)
Comment (joeylnguyen) [reply README.md]: Should be fixed! I noticed a '?' on a few of the other methods as well. Did you want me to clean those up as well?

**Review by Fcmam5 — APPROVED** [review]
Thank you!

**Comment by Fcmam5** [comment]
@all-contributors add @joeylnguyen for code 😃

**Pull request merged**